spec/views/two_factor_authentication/recovery_code_verification/show.html.slim_spec.rb

```diff
@@ -31,4 +31,9 @@
     expect(rendered).
       to have_xpath("//form[@method='post']")
   end
+
+  it 'allows the user to cancel out of the sign in process' do
+    render
+    expect(rendered).to have_link(t('links.cancel'), destroy_user_session_path)
```

Comment:
I threw it into a shared example, only a couple minutes of work and we will probably be able to refactor more tests out into it in the future.